chain/chain/src/store.rs

```diff
@@ -2465,6 +2467,104 @@ impl<'a> ChainStoreUpdate<'a> {
         Ok(())
     }
 
+    /// Only used in mock network
+    /// Copy the necessary chain state related to `block_hash` from `source_store` to the current
+    /// store.
+    #[cfg(feature = "mock_network")]
```

Comment:
This is a hack to make the binary compile without the feature. I can't add this feature as a default feature to integration-tests, because that breaks some tests in integration-tests.

I plan to move mock_network outside of integration-tests to be its own package so I can set mock_network as default feature for the package, but I want to do that in another PR to make review easier